ELSpace/Screens/Hub/Activities/ViewModels/RaportDateProvider.swift

```diff
@@ -0,0 +1,19 @@
+import RxSwift
+import RxCocoa
+import Foundation
+
+protocol RaportDateProviding {
+    var dateRelay: BehaviorRelay<Date> { get }
+}
+
+class RaportDateProvider: RaportDateProviding {
```

Comment:
Correct me if I'm wrong, but do you use this singleton just to change the date in one view model (`ActivitiesViewModel`)? Seems like an overkill?